### ISSUE TITLE
Need to fix Lintian error and warnings

### ISSUE THREAD

**Issue by sriemer** [sriemer/scanmem-debian#4]
```
W: gameconqueror: appstream-metadata-in-legacy-location usr/share/appdata/GameConqueror.appdata.xml
E: gameconqueror: appstream-metadata-legacy-format usr/share/appdata/GameConqueror.appdata.xml
W: gameconqueror: priority-extra-is-replaced-by-priority-optional
W: scanmem: priority-extra-is-replaced-by-priority-optional

E: Lintian run failed (policy violation)
```

**Comment by sriemer** [comment]
Depends on scanmem/scanmem#295.

**Comment by sriemer** [comment]
```
W: gameconqueror: priority-extra-is-replaced-by-priority-optional
W: scanmem: priority-extra-is-replaced-by-priority-optional
```
fixed by f589c0e5e8a7df58f61cb90bde55f66e57062d6c.

Now depending on the upstream bug for
```
W: gameconqueror: appstream-metadata-in-legacy-location usr/share/appdata/GameConqueror.appdata.xml
E: gameconqueror: appstream-metadata-legacy-format usr/share/appdata/GameConqueror.appdata.xml
```

**Comment by sriemer** [comment]
Almost there. Warning fixed on salsa.debian.org. The error will be fixed there.
